content/en/infrastructure/resource_catalog/policies/_index.md

```diff
@@ -57,11 +56,11 @@ Click the new policy to review all non-compliant resources and filter them by re
 Custom policies let you define a target resource attribute and a desired value, providing flexibility in creating policies for your cloud resources without requiring complex query languages. The following features are available:
 
 * **Access data in nested attributes**:
-Validate more of your configurations (for example mandate that `TLS 1.2` stored in a nested property is used for Amazon CloudFront). 
+Validate more of your configurations (for example, require that `TLS 1.2`, which is data stored in a multi-level property, is used for Amazon CloudFront). 
 * **Use advanced condition matching**:
-Use operators like `\>, \<, \!=` (for example enforcing Kubernetes version \> 1.25).  
+Use advanced condition matching: Use operators like `>, <`, or `!=` (for example, enforcing Kubernetes version > 1.25).  
```

Comment:
```suggestion
Use operators like `>`, `<`, or `!=` (for example, enforcing Kubernetes version > 1.25).  
```

Reply:
This is a duplicate of the line above. 

Also, just to confirm -- is it intentional to have the first two operators share the same code block, but the last operator is in its own code block? I re-added the backticks I previously suggested if not, but let me know if there might be some context I'm missing here.

Reply:
Ah so yes this was from the PM, the operator should be `< , >` basically the greater than and less than operator, I don't *think* it matters if they are together or separate but just to be consistent.  I agree one sentence is duplicated. let me check this.

Reply:
Thanks for following up! I did some looking into this, and `<, >` are indeed two **separate** operators. Ideally, for visual clarity and consistency, they should each have their own set of backticks as I suggested above. Screenshot for reference:
<img width="992" alt="operators" src="https://github.com/user-attachments/assets/84b75a1a-059f-46d1-8ec2-2cd55e85c0d3" />

Let me know if you have any feedback!

Reply:
great info!